EIPS/eip-minimal-proxy-push0.md

```diff
@@ -0,0 +1,238 @@
+---
+eip: minimal-proxy-push0
```

Comment:
```suggestion
eip: 7511
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
done, all test have been passed